VideoLocker/src/main/java/org/edx/mobile/third_party/iconify/IconButton.java

```diff
@@ -0,0 +1,64 @@
+package org.edx.mobile.third_party.iconify;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.widget.ImageButton;
+
+import org.edx.mobile.R;
+
+public class IconButton extends ImageButton {
```

Comment:
Curious; why do we need this class? Can't we just use `IconView`? (Not a big deal, just wondering why)